### PR TITLE
Update abac.md

### PR DESCRIPTION
**Pull request by lcfang**
Modified ":" to ";", and added the lost "."

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4968)

### CODE REVIEW

File: docs/admin/authorization/abac.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ properties:
 
   - Versioning properties:
     - `apiVersion`, type string; valid values are "abac.authorization.kubernetes.io/v1beta1". Allows versioning and conversion of the policy format.
-    - `kind`, type string: valid values are "Policy". Allows versioning and conversion of the policy format.
+    - `kind`, type string; valid values are "Policy". Allows versioning and conversion of the policy format.
```
Comment (a-mccarthy) [inline docs/admin/authorization/abac.md]: this bullet is not showing up correctly

**Comment by a-mccarthy** [comment]
@lcfang, can you look into the formatting of your changes? i looks like one of the bullets is no longer showing up correctly

File: docs/admin/authorization/abac.md (hunk 1 above)
Comment (lcfang) [inline docs/admin/authorization/abac.md]: I think I had look into the formatting of my changes, and could you tell me which bullet is not showing up correctly, the ";" or "."?

Comment (a-mccarthy) [inline docs/admin/authorization/abac.md]: `kind` should be a separate bullet from `apiVersion`.  Your changes to ";" and "." are ok, but kind and apiVersion are now showing up as one bullet point on the page

**Comment by zacharysarah** [comment]
@lcfang :wave: Can you please incorporate @a-mccarthy's review?

**Comment by steveperry-53** [comment]
No response from author since August.
@lcfang Please reopen if you want to go forward with this.

**Pull request closed without merging**